Src/Support/Google.Apis.Auth.Tests/Google.Apis.Auth.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <!-- net452 is the earliest desktop version supported by xunit runner -->
+    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
```

Comment:
Any reason for netcoreapp1.1 instead of netcoreapp1.0? Not a problem as far as I'm aware...